applications/sink/rsocket-sink/src/main/resources/META-INF/dataflow-configuration-metadata-whitelist.properties

```diff
@@ -1,2 +1 @@
 configuration-properties.classes=org.springframework.cloud.fn.consumer.rsocket.RsocketConsumerProperties
-
```

Comment:
I don't understand: you removed all others, but then you just fix a new line in this one.
Why don't remove this file as well?

Reply:
That means that there has to be a new file in the RSsocket Sink module instead of this one, like you did for PG Copy.
But we still have discussion with you on a wrong file and probably talk about different things.
Am I missing anything else as a purpose of this PR?

Reply:
Yeh, I got confused on the files. Good catch @artembilan (aka Eagle Eye). It is updated now (stripped the legacy filename ending).